### PR TITLE
Touch up a few tpyos

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Try two, hopefully without merge issues from github.  Touching up a few typos in the README.md as noted in a review earlier today.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
-_Please be aware `skopeo` is still work in progress and it currently supports only registry API V2_
+_Please be aware that `skopeo` is still a work in progress and it currently supports only registry API V2_
```
Comment (rhatdan) [inline README.md]: I think calling skopeo a work in progress denigrates it usefulness.  I believe that we should just have a note saying that it currently support the registry API V2.  What api would we want it to support?  Definitely not V1?

Comment (rhatdan) [reply README.md]: This page should celebrate all that skopeo can do.  We need to add the icon and talk about moving container images between different types of container storages.  Lets pump up Skopeo!

**Comment by mtrmac** [comment]
LGTM, thanks.

(Pumping up skopeo should definitely happen, but does not block this PR.)

File: README.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply README.md]: I concur, but for the short term I'd like to get this one in as the typos need to be cleaned.  I'll work on a better update tomorrow (Friday) morn.

**Pull request merged**